tests/robottelo/test_decorators.py

```diff
@@ -36,20 +39,37 @@ def test_not_smoke(self):
 class BzBugIsOpenTestCase(TestCase):
     """Tests for :func:`robottelo.decorators.bz_bug_is_open`."""
     # (protected-access) pylint:disable=W0212
-    def setUp(self):  # noqa pylint:disable=C0103
+    # (test names are long to make it readable) pylint:disable=C0103
+    def setUp(self):
         """Back up objects and generate common values."""
         self.backup = decorators._get_bugzilla_bug
         self.bug_id = gen_integer()
-
-    def tearDown(self):  # noqa pylint:disable=C0103
+        self.upstream_backup = conf.properties.get('main.upstream')
```

Comment:
This may (actually, I'm almost certain it will) modify `conf.properties` like so:

```python
conf.properties['main.upstream'] = None
```

Can we properly mock out `conf.properties` in the tests? I did the backup/restore thing for quite a while, but now realize that it's a very fragile approach.